server/controllers/conversation.js

```diff
@@ -0,0 +1,60 @@
+const mongoose = require("mongoose");
+const Conversation = require("../models/Conversation");
+const Message = require("../models/Message");
+
+exports.startConversation = async (req, res) => {
+  const { user1, user2 } = req.body;
+
+  const conversationExists = await Conversation.findOne({
+    users: [mongoose.Types.ObjectId(user1), mongoose.Types.ObjectId(user2)],
+  });
+
+  if (conversationExists) {
+    res.status(400).send("Conversation already exists.");
+  } else {
+    const conversation = await Conversation.create({
+      users: [user1, user2],
+    });
+
+    if (!conversation) {
+      res.status(500).send("Unable to start conversation.");
+    }
+
+    res.status(200).send({
+      success: { message: "Conversation started." },
+    });
+  }
+};
+
+exports.loadConversations = async (req, res) => {
+  const { user } = req.body;
```

Comment:
It is not a good practice to send data within a `req.body` in a `GET` request, in this case you can obtain the logged in user's id from the request object - `req.user.id`